src/components/MultiSelect/MultiSelect.scss

```diff
@@ -97,12 +225,6 @@ $dropdown-max-height: 20rem;
   }
 }
 
-.multi-select__dropdown-item-description {
```

Comment:
@petermakowski This didn't seemed to be used anywhere, so I removed it. Is this needed?

Reply:
Good spot. It's no longer used, safe to remove.